src/components/Carousels/QuoteCarousel.tsx

```diff
@@ -7,7 +7,8 @@ import Carousel from 'react-bootstrap/Carousel'
 import { BlockquoteWithLogoBottom } from '@components'
 import { breakpoints } from '@data'
 import { useWindowWidth } from '@hooks'
-interface Blockquote {
+
+interface BlockquoteProps {
```

Comment:
This is probably a good spot to re-use `BlockquoteWithLogoBottomProps` from `Blockquote.tsx`, no? Then we can import it at the top